dim-testsuite/runtest.py

```diff
@@ -383,12 +383,12 @@ def run_test(testfile, outfile, stop_on_error=False, auto_pdns_check=False):
 
 if __name__ == '__main__':
     # start the server process
-
-    parsed =urllib.parse.urlparse(os.getenv('NDCLI_SERVER', 'http://localhost:5000'))
+    parsed = urlparse(os.getenv('NDCLI_SERVER', 'http://localhost:5000'))
     host = parsed.hostname
     port = parsed.port
 
-    server = Popen(['manage_dim', 'runserver', '--port', str(port), '--host', host], stderr=SRVLOG, stdout=SRVLOG)
+    logfile = open(SRVLOG, 'wb+')
+    server = Popen(['manage_dim', 'runserver', '--port', str(port), '--host', str(host)], stderr=STDOUT, stdout=logfile)
```

Comment:
Keeping stderr to the logfile would probably be the better choice here. When running the test cases errors might get lost if they are only printed to stdout.

Reply:
The name is slightly confusing, this is `subprocess.STDOUT` and to quote the upstream documentation:
> Special value that can be used as the stderr argument to Popen and indicates that standard error should go into the same handle as standard output.